src/sources/apache_metrics/mod.rs

```diff
@@ -0,0 +1,394 @@
+use crate::{
```

Comment:
nit: since this module consists of a single file, you can put it in `src/sources/apache_metrics.rs`.

Basically, these two are exactly the same:

* `src/sources/apache_metrics.rs`
* `src/sources/apache_metrics/mod.rs`

The advantage of the second one is if you have multiple modules that you want to namespace under the `apache_metrics` namespace (and potentially re-export the most common public interface at that level).

Reply:
Whoops, never mind, there's actually a `parser.rs` file that was collapsed in the GH UI, so I missed it.

All good 😄